rasa/nlu/config.py

```diff
@@ -106,14 +106,6 @@ def __init__(self, configuration_values: Optional[Dict[Text, Any]] = None) -> No
         if self.__dict__["pipeline"] is None:
             # replaces None with empty list
             self.__dict__["pipeline"] = []
-        elif isinstance(self.__dict__["pipeline"], str):
-            # DEPRECATION EXCEPTION - remove in 2.1
-            raise RasaException(
-                f"You are using a pipeline template. All pipelines templates "
-                f"have been removed in 2.0. Please add "
-                f"the components you want to use directly to your configuration "
-                f"file. {DOCS_URL_MIGRATION_GUIDE}"
-            )
```

Comment:
At some point we have to get rid of this compatibility code. Since pipeline templates are quite old and were deprecated a long time ago, I think it's fine to free our code base from it (they were even deprecated in `1.x`)